public/app/features/logs/components/LogRowMessage.tsx

```diff
@@ -116,6 +117,14 @@ class UnThemedLogRowMessage extends PureComponent<Props> {
     this.props.onToggleContext();
   };
 
+  shouldShowShowContextToggle = (row: LogRowModel | undefined): boolean => {
```

Comment:
Please suggest a better name if you come up with one 🙈

Reply:
Sorry, I wasn't probably clear. I meant, that when we can create just a new variable `const showContextButton = !!showContextToggle?.(row);`  and everywhere where we do `showContextToggle?.(row)` we would do `showContextButton`? But that is nit pick.

Reply:
Gotcha, updated with @matyax suggestion in 4722e506e1639e5351a9932f74f24d72d5d6e67b